docs/language/accounts.mdx

```diff
@@ -211,6 +211,9 @@ to the `prepare` phase of the transaction.
   struct DeployedContract {
       let name: String
       let code: [UInt8]
+
+      // Returns an array of `Type` objects representing all the public struct and resource declarations in this contract
```

Comment:
Is it specifically only structs and resources? It probably also returns enums? Maybe turn the kinds into an example list:
```suggestion
      // Returns an array of `Type` objects representing all the public declarations in this contract (e.g. structs, resources)
```

Maybe also update the docstring accordingly (examples are useful!)